optimum/onnxruntime/quantization.py

```diff
@@ -216,6 +217,53 @@ def fit(
         )
         return self.compute_ranges()
 
+    def apply_smooth_quant(
+        self,
+        dataset: Dataset,
+        save_dir: Union[str, Path],
+        quantization_config: QuantizationConfig,
+        onnx_sq_model_name: Union[str, Path] = "sq_model.onnx",
+        use_external_data_format: bool = False,
+        batch_size: int = 1,
+    ):
+        import importlib
+
+        try:
+            importlib.import_module("neural_compressor.adaptor.ox_utils.smooth_quant")
+        except Exception as e:
+            logging.error(f"{e}.")
+            raise RuntimeError("Neural-compressor is required for SmoothQuant. Please install the library") from e
+
+        import copy
+
+        import onnx
+        from neural_compressor.adaptor.ox_utils.smooth_quant import ORTSmoothQuant
```

Comment:
This is not recommended, reference: https://peps.python.org/pep-0008/#imports

Reply:
This is something done by the ONNXRuntime source code too! Are u suggesting importing it at top level? https://github.com/microsoft/onnxruntime/blob/0f72739b6db129373d221483d61d6637ec11fb28/onnxruntime/python/tools/quantization/quantize.py#L421